tests/author.test.js

```diff
@@ -0,0 +1,162 @@
+        it("Should return 500 if author is ready in the database", async () => {
+          const authorData = authors[0].dataValues;
+          const result = await request(app).post("/authors").send(authorData);
+          expect(result.status).to.equal(500);
+        });
+      });
+      describe("GET /authors", () => {
+        it("gets all author records", async () => {
+          const response = await request(app).get("/authors");
+
+          expect(response.status).to.equal(200);
+          expect(response.body.length).to.equal(3);
+
+          const returnedAuthors = response.body;
+
+          returnedAuthors.forEach((author) => {
+            const fakedAuthor = authors.find((a) => {
+              return a.dataValues.id === author.id;
+            });
+
+            expect(author.title).to.equal(fakedAuthor.title);
+          });
+        });
+      });
+      describe("GET /authors/:id", () => {
+        it("Should get author with a given id", async () => {
+          const author = authors[0].dataValues;
+          const id = authors[0].dataValues.id;
+
+          const result = await request(app).get(`/authors/${id}`);
+          expect(result.status).to.equal(200);
+
+          expect(result.body.author).to.equal(author.author);
+        });
+
+        it("Should return a 404 if the author is not found", async () => {
+          const result = await request(app).get("/authors/999999999999999");
+          expect(result.status).to.equal(404);
+          expect(result.body.error).to.equal("The author could not be found.");
+        });
+      });
+
+      describe("PATCH /authors/:id", () => {
+        it("should update author with the given id", async () => {
+          const newAuthorData = {
+            author: faker.name.findName(),
+          };
+          const id = authors[0].dataValues.id;
+          const result = await request(app)
+            .patch(`/authors/${id}`)
+            .send(newAuthorData);
+
+          expect(result.status).to.equal(200);
+          const updatedAuthor = await Author.findByPk(id);
+          expect(updatedAuthor.dataValues.author).to.equal(
+            newAuthorData.author
+          );
+        });
+
+        it("Should return a 404 when trying to update an author that does not exist", async () => {
+          const newAuthorData = {
+            author: faker.name.findName(),
+          };
+          const result = await request(app)
+            .patch("/authors/99999999999999")
+            .send(newAuthorData);
+
+          expect(result.status).to.equal(404);
+          expect(result.body.error).to.equal("The author could not be found.");
+        });
+      });
+
+      describe("DELETE /authors/:id", () => {
+        it("Should delete an author with the given id", async () => {
+          const id = authors[0].dataValues.id;
+          const result = await request(app).delete(`/authors/${id}`);
+          expect(result.status).to.equal(204);
```

Comment:
As above, please group assertions together at the end of a test, and put all the setup together at the start of the `it` block